src/app.ts

```diff
@@ -0,0 +1,61 @@
+import { MongoDriverFactory } from "./MongoConnectorFactory";
+import { MongoDriver } from "./MongoConnector";
+import * as body from "body-parser";
+import express from "express";
+import { Console } from "console";
+
+const PORT = 3000;
+const app = express();
+
+app.use(body.json());
+
+let taskDataStore;
+MongoDriverFactory.build()
+  .then(async (datastore) => {
+    taskDataStore = datastore;
+  })
+  .catch(e => {
+    throw e;
+  });
+
+  //Gets All Tasks
+app.get("/", (req,res)=> {
+  taskDataStore.listTasks().then((tasks) => {
+    res.send(tasks);
+    console.log(tasks);
+  });
+})
+
+  //Gets specific Tasks via Id
+app.get("/task/:taskId", async (req,res)=> {
+  const task: string = await taskDataStore.readTask(req.params.taskId);
+  res.send(task);
+  console.log(task);
+});
+
+  //Posts req from body
+  //This Works 
+app.post("/", function(req,res){
+  taskDataStore.createTask(req.body).then((task) => {
+    res.send(task);
+    console.log(task +" New Task Was Created!");
```

Comment:
Just to reiterate what Emily wrote, console logs are great for debugging in development, but for anything going into production console logs should be kept to a minimum.